packages/components/src/FilterBar/context/FilterBarContext.tsx

```diff
@@ -81,6 +82,10 @@ export const FilterBarProvider = <ValuesMap extends FiltersValues>({
       onValuesChange({ ...values, [id]: undefined })
     },
     getInactiveFilters: () => getInactiveFilters<ValuesMap>(state),
+    clearAllFilters: () => {
+      dispatch({ type: "clear_active_filters" })
```

Comment:
Technically this action just turns any active removable filters to inactive, but `deactivate_filters` sounded funny. Open to a clearer name if needed 😂

Reply:
`deactivate_filters` sounds fine to me

Reply:
Haha alright! Sold!